api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java

```diff
@@ -39,8 +39,8 @@
       {
         DataAccessLevel.class.getSimpleName(),
         DataAccessLevel.values(),
-        (Function<Short, DataAccessLevel>) StorageEnums::dataAccessLevelFromStorage,
-        (Function<DataAccessLevel, Short>) StorageEnums::dataAccessLevelToStorage
+        (Function<Short, DataAccessLevel>) CommonStorageEnums::dataAccessLevelFromStorage,
```

Comment:
Should fork the test instead

Reply:
I'm working on this. (At the moment, we don't have a test dir in common-api.)

Reply:
(Added common-api test.)